### ISSUE TITLE
Test accessibility before 1.0.0 release

### ISSUE THREAD

**Issue by fatchild** [eprints/creators_roles#7]
Accessibility should be tested using the WAVE plugin in the following areas
- Search if applied
- Workflow (for both user permissions)
- Abstract pages
